src/libmscore/element.h

```diff
@@ -383,9 +383,10 @@ class Element : public ScoreElement
     void setZ(int val) { _z = val; }
 
     int staffIdx() const { return _track >> 2; }
+    void setStaffIdx(int val) { _track = val << 2 | voice(); }
     virtual int vStaffIdx() const { return staffIdx(); }
     int voice() const { return _track & 3; }
-    void setVoice(int v) { _track = (_track / VOICES) * VOICES + v; }
+    void setVoice(int v) { _track = (_track & ~3) | v; }
```

Comment:
Better to use some named constant (as before) instead of magic number

Reply:
Fixed! (in other places too)